sopel/builtins/units.py

```diff
@@ -71,14 +77,16 @@ def temperature(bot, trigger):
 
     if kelvin <= 0:
         bot.reply("Physically impossible temperature.")
-        return
+        return None
 
     bot.say("{:.2f}°C = {:.2f}°F = {:.2f}K".format(
         celsius,
         fahrenheit,
         kelvin,
     ))
 
+    return None
```

Comment:
Hmm, the explicit return here feels weird. No objection to it, but I would probably have left it off.

Reply:
Leaving it out is a mypy error:

```
sopel/builtins/units.py:58: error: Missing return statement  [return]
```